services/speech_to_text/v1.js

```diff
@@ -63,14 +63,20 @@ module.exports = function (RED) {
       }
 
       if (!config.lang) {
-        var message = 'Missing audio language configuration, unable to process speech.';
-        node.error(message, msg)
+        var message_err_lang = 'Missing audio language configuration, unable to process speech.';
+        node.error(message_err_lang, msg)
         return;
       }
 
       if (!config.band) {
-        var message = 'Missing audio band configuration, unable to process speech.';
-        node.error(message, msg)
+        var message_err_band = 'Missing audio band configuration, unable to process speech.';
+        node.error(message_err_band, msg)
+        return;
+      }
+
+      if (!config.continuous) {
+        var message_err_continuous = 'Missing continuous details, unable to process speech.';
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'message_err_continuous' is not in camel case.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2389271691/issues/source?bid=3225115&fileBranchId=3229003#l79)